src/pages/CommentsPage.jsx

```diff
@@ -34,16 +51,33 @@ const CommentsPage = () => {
           {comments.map((comment) => {
             return (
               <li key={comment.comment_id}>
-                <CommentCard {...comment} />
+                <CommentCard comment={comment} setComments={setComments} />
               </li>
             );
           })}
         </ul>
       ) : (
         <p>This article does not have any comments</p>
       )}
-      <CommentForm />
-      <button onClick={handleBackToArticle}>Back to article</button>
+
+      <section id="comment-form">
+        <p>Post a comment</p>
+        {err ? <p>{err}</p> : null}
+        <form onSubmit={handleSubmit}>
+          <label>
+            Comment{" "}
+            <input
```

Comment:
I think you've branched from a branch - im not sure why this is in this commit. Think you're going to have some tricky merge conflicts